test/unit/user_test.rb

```diff
@@ -138,7 +93,7 @@ def setup
     end
   end
 
-  context "try to login" do
+  describe "try to login" do
```

Comment:
Why change context to describe?  I see you're disabling the minispec rails version, but not sure I get why you're adding shoulda-context to add `context`, aliasing `describe` to `context` and then replacing `context` with `describe`.